msg/templates/uorb_microcdr/dds_topics.h.em

```diff
@@ -48,11 +48,11 @@ struct SendTopicsSubs {
 
 	uint32_t num_payload_sent{};
 
-	bool init(uxrSession* session_, uxrStreamId stream_id, uxrObjectId participant_id);
+	bool init(uxrSession* session_, uxrStreamId stream_id, uxrObjectId participant_id, const std::string &client_namespace);
```

Comment:
`std::string` isn't available on embedded.
```suggestion
	bool init(uxrSession* session_, uxrStreamId stream_id, uxrObjectId participant_id, const const char* client_namespace);
```

Reply:
Sure, my bad, I will solve that. cont char* is really not convenient for concatenation but I didn't test embedded afterward.
The binary interface should be nicer too, I will have a look.